### DISCUSSION TITLE
Bug report: 谷歌翻译源 网页翻译不能使用，而谷歌是可以正常访问，谷歌翻译也可以正常访问，也添加相应的地址

### DISCUSSION THREAD

**Discussion by Joyning-cn** [chunibyocola/sc-translator-crx#142] [General]
谷歌翻译显示：链接超时，
而同时：谷歌可以正常访问和登陆，划词谷歌翻译正常，只是网页翻译无法使用
可能是谷歌网页翻译的链接除了问题。

**Comment by chunibyocola** [comment]
是的，这是一个已知的问题，目前已经修复，等待测试完毕后会马上发布新版本。

感谢你的报告。

**Reply by Joyning-cn** [reply]
You are the best!

**Reply by chunibyocola** [reply]
新版本已在 Chrome 商店上发布且已审核通过，更新后问题将会被修复。🙂
